docs/dev/vagrant.rst

```diff
@@ -175,20 +175,14 @@ Upgrading your personal dev VM
 
 If the base VM has been changed (see above), you will want to upgrade your development server. However, upgrading Ubuntu within a virtual machine can cause problems with your database. Therefore, you'll need to export your database, create a new virtual machine, then import your database:
 
-1. Make a copy of `esp/esp/local_settings.py` somewhere with a different name (e.g. on your desktop as "old_local_settings.py"). This file will get overriden by the end of this process and you will want to restore some of the settings from your previous VM setup.
+1. Make a copy of `esp/esp/local_settings.py` somewhere with a different name (e.g. on your desktop as "old_local_settings.py"). The `local_settings.py` file will get overwitten by the end of this process and you will want to restore some of the settings from your previous VM setup.
 
-2. From within the "devsite" folder, run ``git checkout dumpdb``. This branch has the proper code to interact with your current database and create the dump file.
+2. From within the "devsite" folder, run ``fab dumpdb``. This action will save your database as a dump file in the `devsite` folder called `devsite_django.sql`. You can also specify a filename if you would like with ``fab dumpdb:filename``.
 
-3. Run ``vagrant up`` to start the virtual machine.
+3. Run ``vagrant destroy`` (note: this destroys your virtual machine. Only do it once you are sure your database has been backed up and you are ready to continue).
 
-4. Run ``fab dumpdb``. This will save your database as a dump file in the "devsite" folder ("devsite_django.sql"). You can also specify a filename if you would like with ``fab dumpdb:filename``.
+4. Now follow the `VM installation instructions above <https://github.com/learning-unlimited/ESP-Website/blob/main/docs/dev/vagrant.rst#installation>`_, starting at ``vagrant up``.
```

Comment:
We should reference the section of the file without the full path in case the user isn't on the main branch. See https://gist.github.com/TobiasPalludan/5948533e97d260a04ea47e44ea7cc6b0.

Reply:
The link will still work even if you're not on the main branch. Are you worried that the files might differ?

Reply:
Exactly